lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -124,10 +139,18 @@ class ReportRenderer {
    * @return {DocumentFragment}
    */
   _renderReport(report) {
-    const headerStickyContainer = this._dom.createElement('div', 'lh-header-sticky');
-    headerStickyContainer.appendChild(this._renderReportHeader(report));
+    let headerStickyContainer;
```

Comment:
should we rename this now that it's not really the sticky container?

Reply:
done